test/query_test.rb

```diff
@@ -3,15 +3,30 @@
 class QueryTest < Minitest::Test
```

Comment:
Added these tests to check the default page, the page in query params and page in the `get` method